### PR TITLE
From grid to box

### PR DESCRIPTION
**Pull request by cmonez**
Had to change from Grid to flex box because the photo grid was messing with layout. Also added top bar to photos grid. @samasastudio

### CODE REVIEW

File: client/Index.jsx [hunk 1]
```diff
@@ -1,29 +1,66 @@
...
-      .then(({ data }) => {
-        this.setState({ photos: data[0] });
+    // Get all information from all services for top bar and photos
+    Promise.all([
+      fetch(`http://52.14.166.9:3001/api/photos/${this.state.listingId}`),
+      fetch(`http://52.14.166.9:4000/api/description/${this.state.listingId}`),
+      fetch(`http://3.12.169.208:2001/api/location/${this.state.listingId}`),
+      fetch(
+        `http://ec2-3-129-14-177.us-east-2.compute.amazonaws.com:8080/api/reviews/${this.state.listingId}`
+      ),
```
Comment (samasastudio) [inline client/Index.jsx:32]: what are the trade offs for this change to fetch?

File: client/components/TopBar/TopBar.jsx [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    21% 91%,
+    32% 57%,
+    2% 35%,
+    39% 35%
+  );
+  display: inline-block;
+  height: 15px;
+  width: 15px;
+  margin-right: 4px;
+`;
```
Comment (samasastudio) [inline client/components/TopBar/TopBar.jsx:70]: the star!  will be adding this tomorrow.

File: client/Index.jsx (hunk 1 above)
Comment (samasastudio) [inline client/Index.jsx:32]: this needs to be an elastic IP.  hopefully lets set that up tomorrow.

File: client/components/TopBar/TopBar.jsx [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+
+const TopBar = ({ listingInfo }) => {
+  let location, placeName, averageRating, numberOfReviews;
+  if (listingInfo) {
+    location = `${listingInfo.city}, ${listingInfo.country}`;
+    placeName = listingInfo.placeName;
+    averageRating = listingInfo.score;
+    numberOfReviews = listingInfo.reviews;
+    averageRating = listingInfo.score;
+  }
```
Comment (samasastudio) [inline client/components/TopBar/TopBar.jsx:80]: tradeoff here is possibly breaking your display of state, but is good if you're going to make any modifications.

File: client/Index.jsx (hunk 1 above)
Comment (cmonez) [reply client/Index.jsx:32]: Dandy Andy added elastic IPs!

**Pull request merged**